### PR TITLE
lektor: 2.3 -> 3.1.3

### PR DESCRIPTION
**Pull request by das-g**
###### Motivation for this change

version bump (Lektor 3.0 [has been released in 2017](https://pypi.org/project/Lektor/#history), 3.1 in early 2018 and 3.1.3 on 2019-01-26.)

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

Notes:

* The derivation seems to be missing the JavaScript part of Lektor, so editing through the web interface provided by `lektor server` doesn't work. This was already the case with the 2.3 version of the derivation, so isn't a regression of this version bump.
* Upstream [_does_ provide tests](https://github.com/lektor/lektor#want-to-develop-on-lektor), but we don't run them. Should we? If so, can someone help me implement that? If we shouldn't, should we at least remove or change the comment that (falsely) claims "No tests included in archive"?

### CODE REVIEW

File: pkgs/development/python-modules/lektor/default.nix [hunk 1]
```diff
@@ -17,19 +21,20 @@
...
+  version = "3.1.3";
 
-  src = fetchgit {
-    url = "https://github.com/lektor/lektor";
-    rev = "refs/tags/${version}";
-    sha256 = "1n0ylh1sbpvi9li3g6a7j7m28njfibn10y6s2gayjxwm6fpphqxy";
+  src = fetchFromGitHub {
+    owner = "lektor";
+    repo = "lektor";
+    rev = "${version}";
```
Comment (alyssais) [inline pkgs/development/python-modules/lektor/default.nix]:
```suggestion
    rev = version;
```

Comment (alyssais) [inline pkgs/development/python-modules/lektor/default.nix]:
```suggestion
    rev = version;
```

**Review by alyssais — CHANGES_REQUESTED** [review]

**Comment by joachifm** [comment]
@GrahamcOfBorg build pythonPackages.lektor

**Comment by joachifm** [comment]
Superseded by #59353

**Pull request closed without merging**